src/test/java/com/woowacourse/matzip/application/RestaurantServiceTest.java

```diff
@@ -0,0 +1,35 @@
+package com.woowacourse.matzip.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.woowacourse.matzip.application.response.RestaurantResponse;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest
+@Transactional
```

Comment:
```suggestion
@SpringServiceTest
```
후니가 만들어둬서 이거로 사용하시면 될 것 같아요 :)